src/components/PlaidLink/index.native.js

```diff
@@ -1,10 +1,13 @@
-import {useEffect} from 'react';
+import {useEffect, useRef} from 'react';
 import {openLink, useDeepLinkRedirector, usePlaidEmitter} from 'react-native-plaid-link-sdk';
 import Log from '../../libs/Log';
 import CONST from '../../CONST';
 import {plaidLinkPropTypes, plaidLinkDefaultProps} from './plaidLinkPropTypes';
 
 const PlaidLink = (props) => {
+    // We are stashing initial props in a ref since we don't want a new token to trigger the link to open again
+    // and just want openLink() to be called once
+    const propsRef = useRef(props.token);
```

Comment:
Did this come up with the new ESLint rule?

Reply:
Yeah, this one is interesting. We could solve it a couple ways... 

Basically, there is a `props.token` which we would using only once when the component mounts. If we pass `[]` as a dependency array then the exhaustive deps rule will warn that there's a missing dependency. But if we put it in a ref then React knows to ignore it.

We don't want to put the token as a dependency since then it will be "watched" and we will re-trigger `openLink()` if the token changes... I'm not really sure if the token will change, but it would introduce a different behavior.

Another option would be to ignore the rule in this case.

Reply:
Hmmmm, yeah I think I would opt to suppress the exhaustive deps rule with a comment explaining why. NAB though

Reply:
Maybe I just need to get more used to `useRef`. Going to resolve this one too